lib/Core/Memory.h

```diff
@@ -174,14 +174,17 @@ class ObjectState {
 
   uint8_t *concreteStore;
 
-  // XXX cleanup name of flushMask (its backwards or something)
+  // concreteMask[byte] is set if byte is known to be concrete
   BitArray *concreteMask;
 
-  // mutable because may need flushed during read of const
-  mutable BitArray *flushMask;
-
+  // knownSymbolics[byte] holds the symbolic expression for byte,
+  // if byte is known to be symbolic
   ref<Expr> *knownSymbolics;
 
+  // unflushedMask[byte] is set if byte is unflushed
```

Comment:
Those comments are not visible in Doxygen - `///` is.

Reply:
Thanks for the review.  Does Doxygen show comments for private members?

Reply:
In our configuration it does.